client/src/components/Header/SearchBar/TextSearch/TextSearch.js

```diff
@@ -31,16 +31,15 @@ export default class TextSearch extends React.Component {
 
     render() {
         const { searchPlaceholder, searchQuery } = this.state;
-        const { className, onKeyDown, width, barBottomSpacing } = this.props;
+        const { className, onKeyDown } = this.props;
         return (
-            <div className="position-relative" style={{width: width, marginBottom: barBottomSpacing}}>
+            <div className="text-search position-relative">
                 <input type="text"
                        value={searchQuery}
                        onChange={(event) => this.handleChange(event)}
                        placeholder={searchPlaceholder}
                        className={className}
-                       onKeyDown={onKeyDown}
-                       style={{width: '100%'}}/>
```

Comment:
I got rid of all these width: 100%'s because they're usually overkill and can be handled better by making the parent `display: flex`